coremltools/optimize/torch/quantization/_annotation_config.py

```diff
@@ -0,0 +1,109 @@
+#  Copyright (c) 2024, Apple Inc. All rights reserved.
+#
+#  Use of this source code is governed by a BSD-3-clause license that can be
+#  found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+from typing import Optional as _Optional
+
+import torch as _torch
+import torch.ao.quantization as _aoquant
+from attr import define as _define
```

Comment:
coremltools does not have `attr` as a dependency.

Reply:
We are already using it in existing code in `coremltools.optimize.torch` extensively: https://github.com/apple/coremltools/blob/b416f36054af9ca9d10b2d74ba215d0454677ca0/coremltools/optimize/torch/quantization/quantization_config.py#L21

Reply:
It's a dependency also: https://github.com/apple/coremltools/blob/b416f36054af9ca9d10b2d74ba215d0454677ca0/setup.py#L84